modules/EvseManager/README.md

```diff
@@ -0,0 +1,17 @@
+# EvseManager documentation
+
+see `doc.rst`
+
+## Tests
+
+GTest is required for building the test cases target.
```

Comment:
GTest is actually [pulled in as a source-code dependency now](https://github.com/EVerest/everest-core/blob/aed024ba594d9842dacbc25355407c03ba7bffa7/dependencies.yaml#L85) so that comment probably isn't needed anymore